roles/installer/templates/jobs/migration.yaml.j2

```diff
@@ -16,6 +20,7 @@ spec:
             - awx-manage
             - migrate
             - --noinput
+          resources: {{ migration_job_resource_requirements }}
```

Comment:
for users who are upgrading and might not have `migration_job_resource_requirements` defined, can we fallback to using `task_resource_requirements`?

Reply:
Is a fallback required when there's a default for `migration_job_resource_requirements`? Also, there's no definition at all for this job so it could be counted as "new feature".